deposit-app/src/main/resources/cdr-schema.ttl

```diff
@@ -184,4 +185,11 @@ cdr-acl:unitOwner a rdfs:Property ;
     rdfs:label "Unit Owner" ;
     rdfs:domain cdr:AdminUnit ;
     rdfs:range xsd:string ;
-    rdfs:comment "User granted ownership of an administrative unit. Has all access and administrative permissions, as well as permission to permanently destroy objects, create collections, and assign staff permissions. Applies to cdr:AdminUnit objects." .
\ No newline at end of file
+    rdfs:comment "User granted ownership of an administrative unit. Has all access and administrative permissions, as well as permission to permanently destroy objects, create collections, and assign staff permissions. Applies to cdr:AdminUnit objects." .
+
+###### Aspace Properties ######
+cdr-aspace:refId a rdfs:Property ;
```

Comment:
We should add a unit test to verify that the validator permits ref ids on works, and another to verify that it rejects ref ids on a non-work